### PR TITLE
Add javascript to change selfie and to post on the wall

### PR DESCRIPTION
**Pull request by thienantran010**

### CODE REVIEW

File: portfolio/src/main/webapp/wall.html [hunk 1]
```diff
@@ -12,15 +12,19 @@
         <a href="casual.html">Casual</a>
         <a href="wall.html">Wall</a>
     </nav>
-    <label for="wall-input"><h1>Leave a message on the wall: </h1></label>
-    <textarea id="wall-input" name="wall-input"></textarea>
+    <form>
+        <label for="wall-input"><h1>Leave a message on the wall: </h1></label>
```
Comment (aaron-prindle-sps) [inline portfolio/src/main/webapp/wall.html:16]: nit:  Not sure if this is clear but just to double check, the messages added to the wall will only be on the client (local html).  Someone writing on the "wall" will not actually have their message stored and shown to other users or in another session (not propogated into a database, update the served html in any way, etc.)

**Comment by aaron-prindle-sps** [comment]
Can you rebase this onto main or submit a new PR with just the new changes?  Currently this PR has some of the same commits/code from:
https://github.com/thienantran010/software-product-sprint/pull/3

File: portfolio/src/main/webapp/wall.html (hunk 1 above)
Comment (thienantran010) [reply portfolio/src/main/webapp/wall.html:16]: yeah I know. I was hoping I would learn how to implement something like that in the later parts when we cover servlets and apis and stuff

**Comment by thienantran010** [comment]
Hey, I'm not sure how to rebase or submit a new PR because I can't commit anything. When I do git status the terminal says that there's nothing to commit

**Pull request closed without merging**